src/async/variables.ts

```diff
@@ -26,7 +26,7 @@ const transferSelect = ({ runConfiguration }: any) => {
 
 const valueSelect = ({ runConfiguration }: any) => {
   let { point } = runConfiguration
-  const { objective, climate, variables, validRegions } = runConfiguration
+  const { objective, climate, variables, validRegions, customMode } = runConfiguration
```

Comment:
I don't know if we need/want to change this file. We can still fetch variable values, just not show them if custom mode is on. That way we don't need to _re_ fetch values if they turn custom mode on then back off.